Assets/Scripts/Models/HealthSystem.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+
+[Serializable]
+public class HealthSystem
+{
+    //all of this data should probably be generated at runtime or loaded from a file
+    public float Value
+    {
+        get
+        {
+            return _currentHealth;
+        }
+
+        private set
```

Comment:
Since there is no way to change the health outside of this class as is, you should remove the private here